scripts/tmuxinator.sh

```diff
@@ -18,7 +18,7 @@ export FM_VERBOSE_OUTPUT=0
 source scripts/build.sh
 echo "Running in temporary directory $FM_TEST_DIR"
 
-env | sed -En 's/(FM_[^=]*).*/\1/gp' | while read var; do printf 'export %s=%q\n' "$var" "${!var}"; done > .tmpenv
+env | sed -En 's/(^FM_[^=]*).*/\1/gp' | while read var; do printf 'export %s=%q\n' "$var" "${!var}"; done > .tmpenv
```

Comment:
I wonder if there's a difference if `^` is in front or inside the `(...)`.

Reply:
Probably safer to put it outside?

Reply:
Put it outside